share/completions/cjpm.fish

```diff
@@ -0,0 +1,104 @@
+
+# cjpm.fish - Fish completion script for Cangjie Package Manager
+
+# Global options
+complete -c cjpm -l help    -s h -d "Help for cjpm"
+complete -c cjpm -l version -s v -d "Version for cjpm"
+
+# Subcommands
+complete -c cjpm -n "__fish_use_subcommand" -f -a init      -d "Init a new cangjie module"
+complete -c cjpm -n "__fish_use_subcommand" -f -a check     -d "Check the dependencies"
+complete -c cjpm -n "__fish_use_subcommand" -f -a update    -d "Update cjpm.lock"
+complete -c cjpm -n "__fish_use_subcommand" -f -a tree      -d "Display the package dependencies in the source code"
+complete -c cjpm -n "__fish_use_subcommand" -f -a build     -d "Compile the current module"
+complete -c cjpm -n "__fish_use_subcommand" -f -a run       -d "Compile and run an executable product"
+complete -c cjpm -n "__fish_use_subcommand" -f -a test      -d "Unittest a local package or module"
+complete -c cjpm -n "__fish_use_subcommand" -f -a bench     -d "Run benchmarks in a local package or module"
+complete -c cjpm -n "__fish_use_subcommand" -f -a clean     -d "Clean up the target directory"
+complete -c cjpm -n "__fish_use_subcommand" -f -a install   -d "Install a cangjie binary"
+complete -c cjpm -n "__fish_use_subcommand" -f -a uninstall -d "Uninstall a cangjie binary"
+
+
+# 'init' subcommand options
+complete -c cjpm -n "__fish_seen_subcommand_from init" -f -l help       -s h -d "Help for init"
+complete -c cjpm -n "__fish_seen_subcommand_from init" -f -l workspace       -d "Initialize a workspace's default configuration file"
+complete -c cjpm -n "__fish_seen_subcommand_from init" -f -l name       -d "Specify root package name (default: current directory)" -r
+complete -c cjpm -n "__fish_seen_subcommand_from init" -l path       -d "Specify path to create the module (default: current directory)" -r
+complete -c cjpm -n "__fish_seen_subcommand_from init" -f -l type       -d "Define output type of current module" -r -f -a "executable static dynamic"
```

Comment:
looks good; we'll format this file with `fish_indent` before merging, so no need to bother with aligning columns.
(formatting is not currently checked by PR CI but translations are, we should probably either check both or none)